java/query/TypeQLQuery.java

```diff
@@ -109,12 +110,17 @@ public TypeQLMatch.Group.Aggregate asMatchGroupAggregate() {
         }
     }
 
-    protected void appendSubQuery(StringBuilder query, TypeQLToken.Command command, List<?> elements) {
+    protected void appendSubQuery(StringBuilder query, TypeQLToken.Command command, Stream<String> elements, boolean pretty) {
```

Comment:
since this is a very generic method that can't even take a generic, we just pass in the string stream that must be pre-prepared